doc/api/fs.md

```diff
@@ -1767,7 +1767,7 @@ changes:
   * `encoding` {string|null} default = `null`
   * `flag` {string} default = `'r'`
 
-Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
+Synchronous version of [`fs.readFile()`][]. Returns the contents of the `file`.
```

Comment:
Doesn't make sense to me why `file` is highlighted as code even though there is no parameter with that name. I know it's unrelated to this PR, but maybe you would like to fix it while you are at it :)